src/main/java/bio/terra/service/dataset/DatasetTableDao.java

```diff
@@ -135,6 +150,15 @@ private void createColumns(UUID tableId, Collection<Column> columns) {
     }
   }
 
+  private void removeColumns(UUID datasetId, UUID tableId, Collection<Column> columns) {
```

Comment:
Did you mean to move this code?

Reply:
I'm not sure if this was intentional...I'll move it back